### ISSUE TITLE
DataViews.Table generates invalid html with repeating values of id attributes

### ISSUE THREAD

**Issue by triquetra011** [timothycrosley/thedom#4]
example:
&lt;tr&gt;
  &lt;td name="1" id="1"&gt;
  ...
&lt;/tr&gt;
&lt;tr&gt;
  &lt;td name="1" id="1"&gt;
  ...

Ids must be unique across the whole document.  I would suggest setting ids by concatenating a unique table id with unique tr ids with unique th/td ids.  It would be also good to let the user to switch ids off.

**Comment by timothycrosley** [comment]
Agreed! Going to make table autoIDs a property that must be explicitly checked to generate them, in which case the ID will be a combination of the tables id and the header id. This will be included in beta 3.

Thanks!
